src/ShowSourceBundle/Twig/SourceCodeExtension.php

```diff
@@ -49,7 +49,7 @@ public function getFunctions()
 
     public function showSourceCode(\Twig_Environment $twig, \Twig_Template $template)
     {
-        return $twig->render('default/_source_code.html.twig', array(
+        return $twig->render('ShowSourceBundle::_source_code.html.twig', array(
```

Comment:
I'm not sure, but maybe we should rename `_source_code.html.twig` to `source_code.html.twig`. The reason is that from the point of view of the new bundle, this is not a fragment but a "real template" used to display its contents.